Assets/Scripts/View/PlayerCanvas/ProdFabric.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using Management;
+
+public class ProdFabric : MonoBehaviour
+{
+    public Fabric Fabric => GetComponentInParent<Cell>().Fabric;
+    public GameObject Building;
+    public GameObject Simple;
+    public GameObject Upgrade;
+    public GameObject ProductionControl;
```

Comment:
What is it?

Reply:
Fabric states (created in FE objects with these names).

Reply:
And Object-controller production